src/main/java/mil/dds/anet/beans/search/TaskSearchQuery.java

```diff
@@ -34,6 +34,10 @@ public class TaskSearchQuery extends SubscribableObjectSearchQuery<TaskSearchSor
   @GraphQLQuery
   @GraphQLInputField
   private Boolean hasParentTask;
+  // Find tasks that are (not) assigned to one or more Organizations
+  @GraphQLQuery
+  @GraphQLInputField
+  private Boolean isAssigned;
```

Comment:
I would find it slightly more logical if this new field was added before the `taskedOrgUuid` field.
This field should also be added to the `hashCode()` and `equals()` methods.